src/components/navigation/nav/index.js

```diff
@@ -65,8 +66,14 @@ const Nav = () => {
                       {
                         heading: 'Pride',
                         links: [
-                          { title: 'Our story', url: '/about-us' },
-                          { title: 'Our impact', url: '/about-us/impact' },
+                          {
```

Comment:
Do you have the Prettier extension installed in VSCode? If so, head to the file and save it, then should pop back to what it was. These files shouldn't have re-formatted  🙂